docs/source/reference-lowlevel.rst

```diff
@@ -56,6 +56,35 @@ Global statistics
 .. autoclass:: RunStatistics()
 
 
+.. _trio_contexts:
+
+The current Trio context
+------------------------
+
+There are two different types of contexts in :mod:`trio`. Here are the
+semantics presented as a handy table. Choose the right function for
+your needs.
+
++---------------------------------+-----------------------------------+------------------------------------+
+| situation                       | :func:`trio.lowlevel.in_trio_run` | :func:`trio.lowlevel.in_trio_task` |
++=================================+===================================+====================================+
+| inside a running async function | `True`                            | `True`                             |
++---------------------------------+-----------------------------------+------------------------------------+
+| without a running Trio loop     | `False`                           | `False`                            |
```

Comment:
Trio generally avoids calling itself an "event loop" in its documentation. Maybe say "in a thread that has no active call to `trio.run()`" here.